pkg/vpcmodel/nodesConnectivity.go

```diff
@@ -313,6 +304,23 @@ func (v *VPCConnectivity) getPerLayerConnectivity(layer string, src, dst Node, i
 	return result
 }
 
+// load balancer abstraction:
+// currently, AllowedConnsCombined contains the private IPs of the load balancer.
+// the abstraction creates new AllowedConnsCombined,
+// it replaces the private IPs in the with the load balancer itself
+// for each load balancer, it keeps the abstractionInfo, to be used later
+// see details at nodeSetConnectivityAbstraction()
+func (v *VPCConnectivity) abstractLoadBalancers(loadBalancers []LoadBalancer, lbAbstraction bool) {
+	if lbAbstraction {
+		abstractionInfo := newNodeSetAbstraction(v.AllowedConnsCombined)
+		for _, lb := range loadBalancers {
+			ai := abstractionInfo.abstractNodeSet(lb)
```

Comment:
consider renaming `abstractNodeSet` to `abstractLoadBalancer`

Reply:
NodeSetAbstraction is more general - it can work not only on load balancer, but any nodeset that we have.
I didnt made it LB specific so it might be useful later.
this is also the reason for some more comments that you have later